plenum/server/replica.py

```diff
@@ -764,7 +762,7 @@ def processReqDuringBatch(
             self,
             req: Request,
             cons_time: int,
-            discarded_mask: bitarray,
+            idx: int,
             reqs: List,
             rejects: List):
```

Comment:
maybe pass a list of invalid reqs (indices) as input, as was before (invalid_reqs)? It will allow to avoid re-creating of invalid_indices (invalid_indices = [idx for _, idx, _ in rejects])